src/main/java/uk/gov/pay/directdebit/payments/model/GoCardlessEvent.java

```diff
@@ -138,4 +219,139 @@ public GoCardlessOrganisationId getOrganisationIdentifier() {
     public void setOrganisationIdentifier(GoCardlessOrganisationId organisationIdentifier) {
         this.organisationIdentifier = organisationIdentifier;
     }
+
+    public String getDetailsCause() {
```

Comment:
Would be worth considering the use of `Optional` to wrap any optional attributes in the return type of the getters, hopefully the goCardless documentation might tell you which fields are optional. That way it'll be clear to anyone using a `GoCardlessEvent` what is optional.